src/ui/simulator/toolbox/components/datagrid/renderer/connection.cpp

```diff
@@ -241,6 +247,19 @@ bool connectionNTC::cellValue(int x, int y, const Yuni::String& value)
     return Renderer::Matrix<>::cellValue(x, y, value);
 }
 
+IRenderer::CellStyle connectionNTC::cellStyle(int col, int row) const
+{
+    if (!mLoopFlowData)
+        return Renderer::Matrix<>::cellStyle(col, row);
+
+    const double loopFlow = (*mLoopFlowData)[row];
+    const double ntc = Renderer::Matrix<>::cellNumericValue(col, row);
+    if (!checkLoopFlow(ntc, loopFlow))
```

Comment:
pure virtual function, overriden in `connectionNTCdirect` and `connectionNTCindirect`